gubernator/lint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eiu
+export PYTHONPATH=third_party:$GAE_ROOT:$GAE_ROOT/lib/webapp2-2.5.2:$GAE_ROOT/lib/jinja2-2.6
```

Comment:
Fixed.

On Wed, Jul 27, 2016 at 9:32 AM, Joe Finney <notifications@github.com>
wrote:

> In gubernator/lint.sh
> <https://github.com/kubernetes/test-infra/pull/315#discussion_r72474119>:
>
> > +# Copyright 2016 The Kubernetes Authors All rights reserved.
> > +#
> > +# Licensed under the Apache License, Version 2.0 (the "License");
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +#
> > +#     http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +
> > +set -eiu
> > +export PYTHONPATH=third_party:$GAE_ROOT:$GAE_ROOT/lib/webapp2-2.5.2:$GAE_ROOT/lib/jinja2-2.6
>
> Quotes and braces.
>
> export PYTHONPATH="third_party:${GAE_ROOT}:${GAE_ROOT}/lib/webapp2-2.5.2:${GAE_ROOT}/lib/jinja2-2.6"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/315/files/8fe4a09124eee510058b5164e1549fc9c8d32a9d#r72474119>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAM7nAVUkplR8F_lm_3aDka8B74NleaCks5qZ4gLgaJpZM4JVoqy>
> .
>